### PR TITLE
feature: add donor phone number exports

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
Resolves [GIVE-771]

## Description
These changes update the donation and donor exports with a "Donor Phone Number" column.

1. Donor exports will export the donors phone number which can be viewed and edited through the Donor admin screen.
Saved in `give_donor` under `phone`

2. Donation exports will export the Donor Phone Number field made on a donation and can be found in the Donation admin screen under Donor Details. Saved in `give_donationmeta` under `_give_payment_donor_phone`

Reference: https://github.com/impress-org/givewp/pull/7340

## Affects
Donation & Donor exports

## Visuals
https://github.com/impress-org/givewp/assets/75056371/e0fa1b05-95ff-449f-95e1-b0fdeab6c15b

donor-exports

https://github.com/impress-org/givewp/assets/75056371/743ac9f7-b8d2-4db0-b901-af9798a538cc

donation-exports

### Donation Exports
- Create a v3 form
- Add Donor Phone Field block
- Process a donation with the block filled out
- Go to Donations->tools
- Under Export Donation History click Generate csv
- Under Donor Fields ensure Donor's Phone Number is checked
- Click Generate csv
- Verify that the Donor Phone Number column contains a phone number for the donation

### Donor Exports
- Go to the Donor admin screen Donations->Donors & edit a donor 
- On the next screen edit donor again
- Add a phone number to the donor
- Go to Tools screen Donations->Tools &
- Ensure the Donor Phone Number column is checked
- Click Generate csv
- Verify that the Donor Phone Number column contains a phone number for the donor

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-771]: https://stellarwp.atlassian.net/browse/GIVE-771?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: includes/admin/tools/export/give-export-donations-exporter.php [hunk 1]
```diff
@@ -374,6 +378,10 @@ public function get_data() {
 					$data[ $i ]['address_country'] = isset( $address['country'] ) ? $address['country'] : '';
 				}
 
+                if ( ! empty( $columns['phone'] ) ) {
+                    $data[ $i ]['phone'] = $payment_meta['_give_payment_donor_phone'];
+                }
+
```
Comment (glaubersilva) [inline includes/admin/tools/export/give-export-donations-exporter.php:384]: The unreleased tag about this change is missing on the top of this function.

**Review by glaubersilva — APPROVED** [review]
@JoshuaHungDinh Looks good to me. I only left a comment related to a missing unreleased tag, please add that before sending it to QA. Great work!

File: includes/admin/tools/export/give-export-donations-exporter.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply includes/admin/tools/export/give-export-donations-exporter.php:385]: Resolved: db7a057b6c1314642c5aa2330356ae9721f2c47a

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**